scripts/clean.ts

```diff
@@ -87,6 +88,23 @@ async function getGenerated(): Promise<string[]> {
     }
 }
 
```

Comment:
TARS:<br/>Here are some suggestions to address the security vulnerabilities:

1. For API keys and passwords:
   Use environment variables instead of hardcoding sensitive information.
```suggestion
const apiKey = process.env.API_KEY
const password = process.env.PASSWORD
const dbConnection = process.env.DB_CONNECTION
```